sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -76,7 +76,7 @@ The following examples show common scenarios using the `client` [created above](
 
 ### Analyze a conversation
 
-To analyze a conversation, we can then call the `client.AnalyzeConversation()` method which takes the project name, deployment name, and query as parameters.
+To analyze a conversation, we can then call the `client.AnalyzeConversation()` method which takes a conversations project and an utterance as parameters.
```

Comment:
Whenever referring to "conversations" as in the product name, be sure to capitalize it:
```suggestion
To analyze a conversation, we can then call the `client.AnalyzeConversation()` method which takes a Conversations project and an utterance as parameters.
```